.github/workflows/sauce-ie11.yaml

```diff
@@ -42,5 +42,7 @@ jobs:
           SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
           SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
           BROWSERS: preset:sauce-ie11
-        # Retry once automatically.
-        run: npm run test || npm run test
+        run: |
+          export SAUCE_BUILD_ID=lit-tests__${GITHUB_EVENT_NAME}__${GITHUB_SHA}
+          export SAUCE_TUNNEL_ID=${SAUCE_BUILD_ID}_tunnel
```

Comment:
What was being used as the sauce tunnel ID before?

Was the same exact tunnel being used for every invocation of GitHub actions? 

What does Sauce do when it gets two different connections over the same tunnel ID? Does the second attempt to create a tunnel with the ID block until the first tunnel on that ID shuts down? Or does Sauce labs try and disconnect the first one? Or something else?

Reply:
Before only two tunnels were being used. Both push and pull requests for one action would run on a single tunnel.

It's a little difficult to garner _why_ from the logs, but this change greatly reduces the drops with a log similar to `could not connect to proxy, check internet connection` 

It's also a pattern material-web implements in their tests. This change is the minimal env variables to become unique.

Reply:
Ahh, I was trying to find this variable, but couldn't find it anywhere.

It looks like `SAUCE_TUNNEL_ID` is defined in MWC's karma config, here: https://github.com/material-components/material-web/blob/c96cb2e0cbb0e691e7a3d9ae7c7e91d2b59b12e5/karma.conf.js#L125

So, it's not going to have any effect here, right?

Reply:
Exactly, I forgot to include the wtf update in this commit, it's effect should be nil at the moment. But next commit it will be included

Reply:
This won't change how IE times out but it prevents a single tunnel from running multiple GitHub actions (I think currently it affects multiple PRs not just one PR because we all use the same build name)